packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -0,0 +1,170 @@
+import React, {useCallback, useState} from 'react';
+import {convert} from 'css-color-function';
+import {get, getOr} from 'lodash/fp';
+import PropTypes from 'prop-types';
+import Icon from '../../atom/icon';
+import ButtonLink from '../../atom/button-link';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const LearnerSkillCard = (props, context) => {
+  const {
+    'aria-label': ariaLabel,
+    skillCourses,
+    skillQuestions,
+    skillTitle,
+    skillAriaLabel,
+    completedCourses,
+    buttonReviewLabel,
+    buttonReviewAriaLabel,
+    buttonExploreLabel,
+    buttonExploreAriaLabel,
```

Comment:
I think you should go to a translate approach: `translate('courses')` theses should not be injected from exterior.
We won't need to override them.
(skill we do need are these are going to be dynamic)

Reply:
Thank you, I've updated